shared/getting-started/balenaCliAuth/_default.md

```diff
@@ -2,7 +2,8 @@ The recommended way to deploy code is to install the [balenaCLI][cli]. The easie
 
 <img src="/img/common/cli/download-installer.png" width="60%">
 
-__Note:__ You may also install the balenaCLI via npm on a system running NodeJS. In a terminal on your development machine run `npm --global install balena-cli`.
+__Note:__ You may also install the balenaCLI via npm on a system running NodeJS, as explained in
+[NPM Installation][npminstall].
```

Comment:
From my own experience and recent feedback from first-time users in the Hackaton:
* `npm --global install balena-cli` is almost guaranteed to fail on Windows without first installing the dependencies mentioned in the installation instructions, including for example downloading and installing the WDK from Microsoft (plus git, Python, Node, etc.).
* It is also "likely" to fail on Ubuntu and other Linux distros if the system Node is used instead of nvm, as mentioned in the installation instructions.
* `npm --global install balena-cli` duplicates and mismatches the command line given in the installation instructions document, which is `npm install balena-cli --global --production --unsafe-perm`.

The previous getting started wording (before this PR's changes) implied that running "npm install" is likely to succeed, and some Hackaton Windows users did just that, only to get dependency errors.

Reply:
Thanks. Some of this copy was referenced from the CLI masterclass so I'll ensure any instructions match there.